openhub/management/commands/import_organization_data.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from django.core.management.base import BaseCommand
+from openhub.openhub_data import get_organization_data
+from openhub.models import InfographicDetail, Organization
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+class Command(BaseCommand):
+    def import_data(self, org):
+        name = org['name']
+        url = org['url']
+        html_url = org['html_url']
+        created_at = org['created_at']
+        updated_at = org['updated_at']
+        description = org['description']
+        homepage_url = org['homepage_url']
+        org_type = org['type']
+        medium_logo_url = org['medium_logo_url']
+        small_logo_url = org['small_logo_url']
+        outside_committers = org['infographic_details']['outside_committers']
+        outside_committers_commits = org[
+            'infographic_details']['outside_committers_commits']
+        projects_having_outside_commits = org[
+            'infographic_details']['projects_having_outside_commits']
+        portfolio_projects = org['infographic_details']['portfolio_projects']
+        affiliators = org['infographic_details']['affiliators']
+        affiliators_committing_to_portfolio_projects = org[
+            'infographic_details'][
+            'affiliators_committing_to_portfolio_projects']
+        affiliator_commits_to_portfolio_projects = org[
+            'infographic_details']['affiliator_commits_to_portfolio_projects']
+        affiliators_commiting_projects = org[
+            'infographic_details']['affiliators_commiting_projects']
+        outside_projects = org['infographic_details']['outside_projects']
+        outside_projects_commits = org[
+            'infographic_details']['outside_projects_commits']
+        affiliators_committing_to_outside_projects = org[
+            'infographic_details']['affiliators_committing_to_outside_projects']
+
+        try:
+            cr, create = InfographicDetail.objects.get_or_create(
+                outside_committers=outside_committers,
+                outside_committers_commits=outside_committers_commits,
+                projects_having_outside_commits=projects_having_outside_commits,
+                portfolio_projects=portfolio_projects,
+                affiliators=affiliators,
+                affiliators_committing_to_portfolio_projects=affiliators_committing_to_portfolio_projects,
```

Comment:
@jayvdb, How one should fix this? 
Lets say I am doing this
```
affiliators_committing_to_portfolio_projects = \
    affiliators_committing_to_portfolio_projects,
```
Then again when I am running coala it fixes this to this:
```
affiliators_committing_to_portfolio_projects=affiliators_committing_to_portfolio_projects,
```
and then showing the line length error